components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -696,27 +1015,107 @@ private void validateTemplateType(String templateDisplayName, String tenantDomai
         return templateList;
     }
 
+    private void handleServerException(String errorMsg, Throwable ex) throws I18nEmailMgtServerException {
+
+        log.error(errorMsg);
+        throw new I18nEmailMgtServerException(errorMsg, ex);
+    }
+
     /**
-     * Validates the locale of a template
+     * Validate the display name of the notification template.
      *
-     * @param localeCode Locale
-     * @throws I18nEmailMgtClientException Invalid template locale
+     * @param displayName Display name
+     * @throws NotificationTemplateManagerClientException Invalid notification template name
      */
-    private void validateLocale(String localeCode) throws I18nEmailMgtClientException {
+    private void validateDisplayNameOfTemplateType(String displayName)
+            throws NotificationTemplateManagerClientException {
 
-        if (StringUtils.isBlank(localeCode)) {
-            throw new I18nEmailMgtClientException("Locale code cannot be empty or null");
+        if (StringUtils.isBlank(displayName)) {
+            String errorCode =
+                    I18nEmailUtil.prependOperationScenarioToErrorCode(
+                            I18nMgtConstants.ErrorMessages.ERROR_CODE_EMPTY_TEMPLATE_NAME.getCode(),
+                            I18nMgtConstants.ErrorScenarios.EMAIL_TEMPLATE_MANAGER);
+            throw new NotificationTemplateManagerClientException(errorCode,
+                    I18nMgtConstants.ErrorMessages.ERROR_CODE_EMPTY_TEMPLATE_NAME.getMessage());
         }
+        /*Template name can contain only alphanumeric characters and spaces, it can't contain registry invalid
```

Comment:
Use line comments

Reply:
This is a multiline comment. Therefore, using line comments is wrong.